apps/ledger-live-desktop/src/renderer/families/bitcoin/FullNode/steps/StepAccounts.tsx

```diff
@@ -36,7 +37,11 @@ const Accounts = ({
   // FIXME Not using the AccountList component because styles differ quite a bit, we should unify.
   const accounts = useSelector(accountsSelector);
   const currency = getCryptoCurrencyById("bitcoin");
+
   const bitcoinAccounts = accounts.filter(a => getAccountCurrency(a) === currency);
+
+  const unit = useAccountUnit(bitcoinAccounts[0]);
```

Comment:
`[0]` can be falsy so this code seems unsafe? *(btw FullNode is likely untestable because support was dropped but code is still there..)*